### PR TITLE
Migrate Iris PyTorch Notebook to AI Platform samples

### PR DESCRIPTION
**Pull request by gogasca**

### CODE REVIEW

File: notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb [hunk 1]
```diff
@@ -0,0 +1,1202 @@
+{
```
Comment (alecglassford) [inline notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: <p>Can you rephrase this to make it clearer that model.py gets included here, but pytorch gets included separately? maybe:</p><p> </p><pre class="ql-syntax" spellcheck="false">Create a <code>setup.py</code> script to bundle **model.py** in a tarball package. Notice that<code>setup.py</code> does not include the dependencies of <code>model.py</code> in the package. These dependencies are provided to your model version in other ways:

* <code>pandas</code> and <code>google-cloud-storage</code> are both included as part of [AI Platform Prediction runtime version 1.15](https://cloud.google.com/ai-platform/prediction/docs/runtime-version-list).
* <code>torch</code> is provided in a separate package, as described in a following section.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (alecglassford) [inline notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: <p>This cell has a lot of information but is a little hard to follow and has some typos. I think in this case it is ok to just tell the user which PyTorch version they need rather than explaining the whole naming scheme (which seems pretty complicated).  Here is my attempt to simplify (I recommend replacing this whole cell, but feel free to adjust):</p><p> </p><pre class="ql-syntax" spellcheck="false">You need to specify two Python packages when you create your version resource. One of these is the package containing <code>model.py</code> that you uploaded to Cloud Storage in a previous step. The other is a package containing the version of PyTorch that you need.

Google Cloud provides a collection of PyTorch packages in the <code>gs://cloud-ai-pytorch</code> Cloud Storage bucket. These packages are mirrored from the [official builds](https://download.pytorch.org/whl/cpu/torch_stable.html).

For this tutorial, use <code>gs://cloud-ai-pytorch/torch-1.3.1+cpu-cp37-cp37m-linux_x86_64.whl</code> as your PyTorch package. This provides your version resource with PyTorch 1.3.1 for Python 3.7, built to run on a CPU in Linux.

Use the following command to create your version resource:
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

**Comment by alecglassford** [comment]
I didn't get to go through super carefully but I left a couple of recommendations. I will try to take a closer look when I get a chance.

File: notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb (hunk 1 above)
Comment (gogasca) [reply notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: <p>ack</p>

Comment (gogasca) [reply notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: <p>ack</p>

**Comment by gogasca** [comment]
<p>Thanks Alec</p>

**Comment by gogasca** [comment]
@sirtorry
@thedriftofwords ping

File: notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb [hunk 2]
```diff
@@ -0,0 +1,1195 @@
...
+    "\n",
+    "### Costs \n",
+    "\n",
+    "This tutorial uses billable components of Google Cloud Platform (GCP):\n",
+    "\n",
+    "* Cloud AI Platform\n",
+    "* Cloud Storage\n",
+    "\n",
+    "Learn about [Cloud AI Platform\n",
+    "pricing](https://cloud.google.com/ml-engine/docs/pricing) and [Cloud Storage\n",
```
Comment (alecglassford) [inline notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: Can you update this link to `https://cloud.google.com/ai-platform/prediction/pricing`? And the link text to `[AI Platform Prediction] pricing?

(Currently link redirects to Training pricing)

File: notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb [hunk 3]
```diff
@@ -0,0 +1,1195 @@
...
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table align=\"left\">\n",
+    "  <td>\n",
+    "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/ai-platform-samples/blob/master/notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb\"\">\n",
```
[the remark this answers is not in the document]
Comment (gogasca) [reply notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: ack

File: notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb (hunk 2 above)
Comment (gogasca) [reply notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb]: ack

**Comment by gogasca** [comment]
> Two small suggestions for small issues. Otherwise it looks pretty good to me.
> 
> If/when we publish a version of this on cloud.google.com, there are a few additional grammar/branding/style changes I would suggest - but I don't want to block on these right now. I am happy to revisit some time in the future if you remind me.

Sounds good, thanks @alecglassford
